apps/site/pages/courses/[courseSlug]/index.tsx

```diff
@@ -183,6 +185,15 @@ export default function Course({ course, summary, content, markdownDescription }
 	);
 }
 
+// Checks if user is logged in.
+function useSessionInfo() {
```

Comment:
non suiting method name. Suggestion is a more descriptive method name like isUserAuthenticatedInSession()